src/SFA.DAS.EmployerAccounts.Web/Views/Transfers/TransferConnectionInvitationAuthorization.cshtml

```diff
@@ -8,7 +8,7 @@
 
 @if (Model.AuthorizationResult.IsAuthorized)
 {
-    <p>Before starting a connection, both the sending and receiving employers need to read and understand the <a href="https://www.gov.uk/government/publications/apprenticeship-funding-and-performance-management-rules-2017-to-2018" class="govuk-link" rel="nofollow">rules for sending and receiving transfers</a>.</p>
+    <p>Before starting a connection both the sending and receiving employers need to read and understand the <a href="https://www.gov.uk/government/publications/apprenticeship-funding-and-performance-management-rules-2017-to-2018" class="govuk-link" rel="nofollow">rules for sending and receiving transfers</a>.</p>
```

Comment:
I know its in the AC but does is this correctly punctuated without the comma?

Reply:
I've just asked an English teacher and she says that both are technically correct but that she prefers the green one